src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Grammar.Test/.npmrc

```diff
@@ -1,2 +1,3 @@
 # Auto generated file from Gardener Plugin CentralFeedServiceAdoptionPlugin
 registry=https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public-npm/npm/registry/
+always-auth=true
```

Comment:
This is the magic that allowed me to auth, and auth is the reason dependabot couldn't update this package, so maybe this also fixes dependabot? I doubt it though 😛